website/pages/api-docs/jobs.mdx

```diff
@@ -558,6 +565,9 @@ The table below shows this endpoint's support for
 
 ### Parameters
 
+- `diffs` `(bool: false)` - Specifies if the Diffs field should be populated,
+  containing the difference between the current and last job version.
```

Comment:
Maybe something like "containing the structured diff between the current and last job"? Something to make it clear that the diff here is a JSON object from which a pretty-printed diff like seen in the CLI can be constructed.

Reply:
I think the tricky bit is that the returned `Diffs` are an array, that must be inferred by index which version it is a diff of :/ Your suggestion is a good improvement, its definitely a tricky response